src/core/test-rules.rkt

```diff
@@ -48,4 +48,5 @@
 (module+ test
   (for* ([rule (in-list (*rules*))])
     (test-case (~a (rule-name rule))
-      (check-rule rule))))
+      (when (set-member? (rule-tags rule) 'sound)
```

Comment:
We're skipping unsound rules because they have these new unsound-xxx operations in them now?